### DISCUSSION TITLE
Laser only cuts pieces of a line

### DISCUSSION THREAD

**Discussion by nichtAlex** [arkypita/LaserGRBL#1027] [Q&A]
I want to cut a line from P1 to P2. My code is (starting from P1[X187.5 / Y150]) "G1 X157.5 Y120" which should cut a solid line but somehow ends up only cutting half of it. The laser is toggled on all the time while moving from P1 to P2. On the left side of the picture are also a few lines missing.
![rect](https://user-images.githubusercontent.com/57851179/83311005-2f98b400-a20e-11ea-9314-7b60fdd7046e.png)
Any ideas why that could happen?
My laser should be 6000mW. PSU is 12V 10A. Im using an octocuppler (i think) to step the 5V PWM signal down to 3.3V because thats what my laser needs.
This is how its supposed to look. I made a sketch to help me write the code.
![how its supposed to look](https://user-images.githubusercontent.com/57851179/83311333-2f4ce880-a20f-11ea-932f-6d477bc703a9.png)

**Comment by gmmanonymus111** [comment]
This is caused by the reflectiveness of the white material you try to cut. The laser can't 'bite' the material, but once it burned, the material turns dark, and the laser can cut the rest of it easily.

Try with lower speed or higher laser power (or use a darker non-reflective material).
